apps/condo/domains/common/utils/pdf.ts

```diff
@@ -1,33 +1,71 @@
 import html2canvas from 'html2canvas'
 import Jspdf from 'jspdf'
 
+const PDF_FORMAT_SETTINGS = {
+    // pdfWidth - width of final image (in mm)
+    // pdfHeight - height of final image (in mm)
+    // elementOffset - offset of image (in mm)
+    // firstLineOffset - margin right for first line (in css pixels)
+    // lineSpace - margin right for rest of lines (in css pixels)
+    'a4': { pdfWidth: 210, pdfHeight: 297, elementOffset: 10, firstLineOffset: 23, lineSpace: 80, ratio: Math.sqrt(2) },
```

Comment:
Maybe we can move `elementOffset`, `firstLineOffset` and `lineSpace` to different constants? They seems to me equal in all cases

Reply:
`ratio` too)

Reply:
> Maybe we can move `elementOffset`, `firstLineOffset` and `lineSpace` to different constants? They seems to me equal in all cases

`elementOffset` telling us what distance to keep for "safe printing" (some printers cannot print edges correctly, so it may vary....here's just test values of 10 (mm) for both formats, but it can change for others I guess)

`firstLineOffset` and `lineSpace` tell us how far are lines from each other....right now it's proportional (that's why equal, but I'm sure it will change later)

Reply:
> `ratio` too)

already removed, now using `pdfHeight` instead to calculate the rest

Reply:
But `ratio` is still here :(

Reply:
oops, removed redundant const